build/ci/job-template.yml

```diff
@@ -10,7 +10,7 @@ parameters:
 jobs:
   - job: ${{ parameters.name }}
     ${{ if eq(parameters.codeCoverage, 'false') }}:
-      timeoutInMinutes: 60
+      timeoutInMinutes: 75
```

Comment:
We really need to get a handle on this situation. Our build times have doubled in the past week.